dependencies.yaml

```
@@ -241,17 +242,23 @@ dependencies:
       - output_types: conda
         packages:
           - &cupy_unsuffixed cupy>=13.6.0
+          - libnvimgcodec == 0.6.0
+          - pip
+          - pip:
+              - nvidia-nvimgcodec-cu12 >= 0.6.0
```

The Python package in Conda is `nvimgcodec`. So we can just use that here


```suggestion
          - nvimgcodec>=0.6.0
```